inference-engine/src/gna_plugin/frontend/scale_factor_calc.hpp

```diff
@@ -205,7 +205,7 @@ class ScaleFactorPerLayer {
      * @param result
      * @return
      */
-    bool operator()(T cnnLayer, int weightsSize, int inputsSize, ScaleFactorUpdateResult &result, const bool fakeQuantize) {
+    bool operator()(T cnnLayer, int weightsSize, int inputsSize, ScaleFactorUpdateResult &result, bool fakeQuantize, int isInfiniteLoopCount) {
```

Comment:
isInfiniteLoopCount name seems to be a bit confused for int attribute. What does 'is' prefix mean here?

Reply:
renamed